### ISSUE TITLE
Move / Refactor Media Player Helpers

### ISSUE THREAD

**Issue by 12** [bbc/simorgh#2720]
_This will be blocked on #2614_

**Is your feature request related to a problem? Please describe.**
Creating this as a new issue instead of adding it to #2614 to reduce size of PRs, and to not 'feature-creep'. 

The `AudioVideo/index.jsx` file would look a lot tidier if we moved some logic out of the index file. 

**Describe the solution you'd like**
Consider how the following can be refactored:
https://github.com/bbc/simorgh/blob/dd74047a83575eaab569167f11c3d07f7ea331a5/src/app/containers/AudioVideo/index.jsx#L47-L72

With the new AV implementation in mind, I think we need just the following values to pull the player into the page:
1. Asset ID
2. Version ID of media asset to display
3. Platform (amp/canonical)
4. Environment (we will need to change envs of the embed endpoint to match our data source env)
5. Orientation (as we need to adapt the grid, like we currently do)

**Describe alternatives you've considered**

**Testing notes**
[Tester to complete]

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

**Additional context**
